procrastinate/admin.py

```diff
@@ -47,39 +48,44 @@ async def list_jobs_async(
             Filter by job status (*todo*/*doing*/*succeeded*/*failed*)
         lock : ``str``
             Filter by job lock
+        queueing_lock : ``str``
+            Filter by job queueing_lock
 
         Returns
         -------
         ``List[Dict[str, Any]]``
             A list of dictionnaries representing jobs (``id``, ``queue``, ``task``,
             ``lock``, ``args``, ``status``, ``scheduled_at``, ``attempts``).
         """
-        return sorted(
-            [
-                {
-                    "id": row["id"],
-                    "queue": row["queue_name"],
-                    "task": row["task_name"],
-                    "lock": row["lock"],
-                    "args": row["args"],
-                    "status": row["status"],
-                    "scheduled_at": row["scheduled_at"],
-                    "attempts": row["attempts"],
-                }
-                for row in await self.connector.execute_query_all(
-                    query=sql.queries["list_jobs"],
-                    id=id,
-                    queue_name=queue,
-                    task_name=task,
-                    status=status,
-                    lock=lock,
-                )
-            ],
-            key=lambda x: x["id"],
-        )
+        return [
+            {
+                "id": row["id"],
+                "queue": row["queue_name"],
+                "task": row["task_name"],
+                "lock": row["lock"],
+                "queueing_lock": row["queueing_lock"],
+                "args": row["args"],
+                "status": row["status"],
+                "scheduled_at": row["scheduled_at"],
+                "attempts": row["attempts"],
+            }
+            for row in await self.connector.execute_query_all(
+                query=sql.queries["list_jobs"],
+                id=id,
+                queue_name=queue,
+                task_name=task,
+                status=status,
+                lock=lock,
+                queueing_lock=queueing_lock,
+            )
+        ]
 
     async def list_queues_async(
-        self, queue: str = None, task: str = None, status: str = None, lock: str = None,
+        self,
+        queue: Optional[str] = None,
+        task: Optional[str] = None,
+        status: Optional[str] = None,
+        lock: Optional[str] = None,
```

Comment:
Doesn't this one need a `queueing_lock` parameter as well?

Reply:
I'm not sure it makes sense... What information would you be searching if you listed queues with a queueing_lock filter ?

Happy to add it if you have a use case or if you think the lack of consistency in this aspect would be confusing.

Reply:
No it was just for the consistency with `list_jobs`.